core/src/text/textBuffer.cpp

```diff
@@ -213,26 +229,29 @@ bool TextBuffer::addLabel(const TextStyle::Parameters& _params, Label::Transform
     uint32_t strokeWidth = _params.strokeWidth / _params.blurSpread * 255. * .25;
     uint32_t stroke = (_params.strokeColor & 0x00ffffff) + (strokeWidth << 24);
 
-    FontContext::FontMetrics metrics = _fontContext.getMetrics();
+    m_metrics = _fontContext.getMetrics();
+    m_bbox = glm::vec2(0);
 
     /// Apply word wrapping
-    glm::vec2 bbox;
-    std::vector<TextBuffer::WordBreak> wordBreaks;
-    int nLine = applyWordWrapping(quads, _params, _fontContext.getMetrics(), _type, wordBreaks);
+    int nLine = 1;
+
+    m_wordBreaks.clear();
+    nLine = applyWordWrapping(quads, _params, m_metrics);
 
     /// Generate the quads
     float yMin = std::numeric_limits<float>::max();
     float xMin = std::numeric_limits<float>::max();
 
+    m_scratchVertices.clear();
```

Comment:
Sure, would be ok to avoid possible over-allocations or multiple reallocations on push_back below. Though I generally took the approach to allow reused buffers to over-allocate and overall this should reduce reallocations to happen in relatively few runs.